src/components/QuickOpenModal.js

```diff
@@ -434,11 +435,9 @@ function mapStateToProps(state) {
 export default connect(
   mapStateToProps,
   {
-    shortcutsModalEnabled: actions.shortcutsModalEnabled,
```

Comment:
These were unused. It's easy to overshadow DispatchProps, with OwnProps. The current typing situation doesn't guard against overshadowing props with multiple things, but it does check for improper type usage.